app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class ReservationsController < ApplicationController
+  def index
+    reservations = Reservation.all
+    render json: reservations
+  end
+
+  def show
+    reservation = Reservation.find(params[:id])
+    render json: reservation
+  end
+
+  def create
+    reservation_db = Reservation.joins(:screening)
+    reservation = reservation_db.new(reservation_params)
+    byebug
```

Comment:
you left byebug

Reply:
right, i forgot to run rubocop altogether